server/controller/getQuestions.js

```diff
@@ -0,0 +1,20 @@
+const getQuestionsQuery = require('../database/queries/getQuestionsQuery');
+
+const getQuestions = (req, res) => {
+	getQuestionsQuery()
+		.then((result) => {
+			if (!result) {
+				return res
+					.status(500)
+					.send({ message: 'Couldnt get the questions from the db' });
+			}
+			res.status(200).send(result.rows);
+		})
+		.catch((err) =>
+			res
```

Comment:
generally, handling server errors is best done via the `next` function.

Reply:
Also, the message should be `internal server error`.  Most errors have standard messages. Use them.